### PR TITLE
Check labels annotations filter

### PR DESCRIPTION
**Pull request by yuriolisa**
**Description:** <Describe what has changed.>

Fixing the Labels and Annotations filter.

**Link to tracking Issue(s):** <Issue number if applicable>

- Resolves: #2770 
**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: internal/manifests/manifestutils/labels.go [hunk 1]
```diff
@@ -25,8 +25,9 @@ import (
 )
 
 func IsFilteredSet(sourceSet string, filterSet []string) bool {
```
Comment (jaronoff97) [inline internal/manifests/manifestutils/labels.go:27]: do we need to compile the pattern here? Is it not enough to just match string on it?

Comment (yuriolisa) [reply internal/manifests/manifestutils/labels.go:27]: Theoretically, we shouldn't, but without doing that, the regexp match doesn't work as expected.

**Review by jaronoff97 — APPROVED** [review]

**Comment by pavolloffay** [comment]
@yuriolisa the PR needs to be rebased

**Review by pavolloffay — APPROVED** [review]

**Review by swiatekm-sumo — APPROVED** [review]

File: internal/manifests/manifestutils/labels.go (hunk 1 above)
Comment (swiatekm-sumo) [reply internal/manifests/manifestutils/labels.go:27]: What is the difference between them?

**Comment by Intuinewin** [comment]
Hi, the `webhook-port` flag was removed in this PR !!!

**Pull request merged**